### PR TITLE
[ELY-2684] Unit test code refactor (removed duplicates)

### PR DESCRIPTION
**Pull request by sugan0tech**
https://issues.redhat.com/browse/ELY-2684

### CODE REVIEW

File: tests/base/src/test/java/org/wildfly/security/sasl/otp/OTPTest.java [hunk 1]
```diff
@@ -1082,6 +994,40 @@ private CallbackHandler createClientCallbackHandler(String username, String pass
         return ClientUtils.getCallbackHandler(new URI("remote://localhost"), context);
     }
 
+    private void testAuthentication(String seed, int sequenceNumber) throws Exception {
```
Comment (fjuma) [inline tests/base/src/test/java/org/wildfly/security/sasl/otp/OTPTest.java]: Would be good to rename this to explain what it does, e.g., checkPasswordRemainsUnchanged or testPasswordRemainsUnchanged, etc.

**Review by fjuma — CHANGES_REQUESTED** [review]
Looks good, @sugan0tech, I've just added a small comment.

For the title and commit message, it would be good to include information about the specific test that was updated, e.g., [ELY-2648] Update tests in OTPTest to use a common helper method for tests where the password should remain unchanged"

File: tests/base/src/test/java/org/wildfly/security/sasl/otp/OTPTest.java (hunk 1 above)
Comment (sugan0tech) [reply tests/base/src/test/java/org/wildfly/security/sasl/otp/OTPTest.java]: Sure :+1:

**Review by fjuma — APPROVED** [review]
Thanks very much @sugan0tech!

**Review by PrarthonaPaul — APPROVED** [review]
LGTM! 
Thanks @sugan0tech

**Review by skyllarr — APPROVED** [review]

**Pull request merged**